react/week2/AddToDo.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+//bootstrap styling
+import { Button } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+export default function AddToDo(props) {
+  const { onAddToDo } = props;
+
+  const handleButtonClick = () => {
+    const newToDo = document.getElementById('newToDo');
+    if (newToDo) {
+      onAddToDo(newToDo.value);
+    }
+  };
+
+  return (
+    <div>
+      <input type="text" id="newToDo"></input>
+      <Button onClick={handleButtonClick}>Add ToDo</Button>
+    </div>
```

Comment:
1. You could pass `setTodos` to this component as a prop and use it to define `onAddTodo` function inside here, so that it's self contained. Alternatively, define it in the parent component and pass the function as a prop. Just pick one (since now you're both importing a function and defining `handleButtonClick`) in order to keep the logic in one place. 👍 


2. If you replaced this div with a form element, you could also submit a todo by pressing enter and it would be more in line with HTML standards: 

```
    <form onSubmit={onAddTodo}>
      <label>
        Add a todo:
        <input
          value={input}
          onChange={(e) => setInput(e.target.value)}
        />
      </label>
      <button type="submit">Submit</button>
    </form>
```

This would also require: 

1. Using `useState` inside this component to track input value with a hook
2. Adding `e.preventDefault()` to the `onAddTodo` function in order to prevent the form from refreshing the page (and of course passing `e` to `onAddTodo`)